chain/chain-primitives/src/error.rs

```diff
@@ -146,6 +146,9 @@ pub enum Error {
     /// `next_bps_hash` doens't correspond to the actual next block producers set
     #[error("Invalid Next BP Hash")]
     InvalidNextBPHash,
+    /// The block has a protocol version that's outdated
```

Comment:
let's also update CHANGELOG.md

Reply:
Do I add the feature in CHANGELOG.md when we stabilize it?